tests/e2e/specs/modules/tagmanager/setup.test.js

```diff
@@ -206,7 +206,10 @@ describe( 'Tag Manager module setup', () => {
 			// Ensure expected tag is placed.
 			await Promise.all( [
 				page.goto( createURL( '/' ) ),
-				page.waitForNavigation(),
+				page.waitForNavigation( {
+					waitUntil: 'networkidle2',
+					timeout: 0,
```

Comment:
Is there something special about this combination? Looks like there is a `waitForNetworkIdle` method as well although this may not be available until newer versions.

Reply:
Very valid question, @aaemnnosttv!

To be very honest, I did quite a lot of exploration but couldn’t find a definitive reasoning behind this combination working. It was the result of a fair bit of trial and error. I also noticed the `waitForNetworkIdle` method, but yeah, to use that we’d need to update Puppeteer.

On that note, I opened #5862 so that we can explore and try to update Puppeteer to the latest version.

Thank you!

Reply:
Thanks! That works for me, just thought I'd ask 😄